Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml

```diff
@@ -2,6 +2,20 @@
 @using CO.CDP.OrganisationApp.Constants
 @using Microsoft.AspNetCore.Mvc.Localization
 @using CO.CDP.Localization
+
+@functions {
+    private async Task<UserInfo?> GetUserInfo()
+    {
+        try
+        {
+            return await userInfoService.GetUserInfo();
+        }
+        catch
+        {
+            return null;
+        }
+    }
+}
```

Comment:
Is this supposed to be here? And this line a bit further down:

```
var userInfo = User.Identity?.IsAuthenticated == true ? await GetUserInfo() : null;
```

Feels like the kind of thing that could do with being reusable rather than sitting in Layout.cshtml?

Reply:
Yes, it didn't feel right when I wrote it. Last-quick-Friday-fix.

I think the real problem is we use something that throws an exception in the template. My mistake was removing this: https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/pull/965/commits/2979a429d3101a55c263e65f9ee5a1a3bb2f29c2#diff-57befefa70a93ad3b3a0d46a4ffa3f9c79456991ce8406d70ae022a7d22b56e1

Reply:
Brought back `HasTenant` as `HasOrganisations`